conduct.md

```diff
@@ -100,13 +100,13 @@ TODO: Write an introductory paragraph.
 
 ### Postmortems
 
-TODO: Write an introductory paragraph.
+Postmortems are evaluation of performances after implementation. The goal is to identify what worked and what didn't and describe what needed to be done differently. When a postmortem is done correctly all team members should have a better understanding of how to avoid the same mistakes in future projects.
```

Comment:
In the first sentence "Postmortems are evaluation of performances", should this have been "are an evaluation" or "are evaluations" instead?